src/engine/source/indexerconnector/include/indexerConnector/indexerConnector.hpp

```diff
@@ -66,27 +90,17 @@ class EXPORTED IndexerConnector final : public IIndexerConnector
      * bulk either when the maximum bulk size or the time interval is reached. The bulk size is 1000 messages and the
      * interval is 5 seconds.
      *
-     * @param config Indexer configuration, including the index name, server list, ssl configuration and user and
-     * password.
-     * @param timeout Interval for monitoring the server health.
-     * @param workingThreads Number of working threads used by the dispatcher. More than one results in an unordered
-     * processing.
-     * @note Example of the configuration:
-     *  {
-     *      "name": "wazuh-alerts-5.x",
-     *      "host": ["localhost:9200"],
-     *      "user": "admin",
-     *      "password": "admin",
-     *      "ssl": {
-     *          "certificate_authorities": "/etc/ssl/certs/ca.pem",
-     *          "certificate": "/etc/ssl/certs/cert.pem",
-     *          "key": "/etc/ssl/certs/key.pem"
-     *      }
-     *  }
+     * @param config Indexer configuration includes:
+     *  - Index name .
+     *  - Server list .
+     *  - Ssl configuration (cacert, cert, and key).
+     *  - Authentication (username and password).
+     *  - Timeout (Interval for monitoring the server health).
+     *  - Working threads number (Number of working threads used by the dispatcher. More than one results in an
```

Comment:
It is not necessary to clarify the documentation of the sub-parameters since they are actually in the structure and that is the source of the data.

```suggestion
     * @param config Indexer connector configuration 
```